hera_qm/utils.py

```diff
@@ -635,9 +687,42 @@ def or_collapse(a, weights=None, axis=None, returned=False):
         return o
 
 
+def and_collapse(a, weights=None, axis=None, returned=False):
```

Comment:
This function allows us to convert a flag array to waterfall by flagging time/freq/pol pixels that are completely flagged.

Reply:
I question the need for this function. On one hand, it's symmetric with with other steps. However the guts of it are basically just one line, `o = np.all(a, axis=axis)`. In the interests of code maintainability and avoiding bloat, I'd replace it in the code with the numpy equivalent.